packages/preset/src/shortcuts/card.ts

```diff
@@ -1,18 +1,23 @@
 export const cardShortcuts: Record<string, string> = {
   // card
-  'o-card-bg': 'bg-gradient-linear bg-gradient-from-rgba(255, 255, 255, 0.2) bg-gradient-to-rgba(255, 255, 255, 0.035) backdrop-blur-lg',
-  'o-card-base': 'o-card-bg o-transition list-none relative rounded-md c-context w-full of-hidden',
+  'o-card-bg':
+    'bg-gradient-linear bg-gradient-from-rgba(255, 255, 255, 0.2) bg-gradient-to-rgba(255, 255, 255, 0.035) backdrop-blur-lg',
+  'o-card-base':
+    'o-card-bg o-transition list-none relative rounded-md c-context w-full of-hidden',
```

Comment:
I like to align other lines without a new line.

Reply:
Caused by vscode auto format 😲.